client-v2/src/components/FeedSection.tsx

```diff
@@ -9,27 +9,30 @@ import FeedSectionHeader from './FeedSectionHeader';
 
 const FeedSectionContainer = styled('div')`
   background-color: ${({ theme }) => theme.shared.base.colors.backgroundColor};
+  width: 100%;
 `;
 
 const FeedWrapper = styled('div')`
-  width: 409px;
+  flex: 3 1 20vw;
```

Comment:
ooh `vw` units. fancy!

Reply:
I'm not sure what prompted such a fancy unit - I think it's the first time I've ever used it 👌